tox.ini

```diff
@@ -1,16 +1,30 @@
 [tox]
-envlist = py36, py37, py38, py39, pypy, pypy3
-skip_missing_interpreters=true
+envlist = py36, py37, py38, py39, py310, pypy3, type
+skip_missing_interpreters = true
 
 [testenv]
 whitelist_externals = git
 deps =
     -rtest-requirements.txt
+passenv =
+    TERM
```

Comment:
What does this do?

Reply:
This allows mypy to detect that color is supported by the running terminal (well, assuming it is), and causes it to print colored output instead of uncolored output.

Reply:
Nox doesn't filter environment variables, but Tox does. TERM shouldn't ever be filtered, since tools might check it to see what you are running in (like mypy does).